### PR TITLE
PD-129: Create Emotion Package

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

Created a separate package to house internal instance of emotion, and break it out from lib. Removed emotion as a dependency of lib in favor of @leafygreen-ui/emotion as a lib dependency.

Tested in storybook, and all tests still passing!

🎟 _Jira ticket:_ [PD-129](https://jira.mongodb.org/browse/PD-129)

## 🛠 Types of changes

Internal tooling update

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: packages/Checkbox/src/Checkbox.story.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { storiesOf } from '@storybook/react';
 import { text, boolean } from '@storybook/addon-knobs';
-import { emotion } from '@leafygreen-ui/lib';
+import emotion from '@leafygreen-ui/emotion';
```
Comment (DesignerDave) [inline packages/Checkbox/src/Checkbox.story.tsx]: Looks like not destructuring before was just out of convenience here. We can pull `css` out for this file.

File: packages/RadioBoxGroup/src/RadioBoxGroup.story.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { text, boolean, select } from '@storybook/addon-knobs';
-import { emotion } from '@leafygreen-ui/lib';
+import emotion from '@leafygreen-ui/emotion';
```
Comment (DesignerDave) [inline packages/RadioBoxGroup/src/RadioBoxGroup.story.js]: Ditto here - let's pull out the method(s) we need from emotion wherever possible.

File: packages/emotion/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@leafygreen-ui/emotion",
+  "version": "0.9.0",
+  "description": "leafyGreen UI Kit Emotion Library",
```
Comment (DesignerDave) [inline packages/emotion/package.json]: Maybe "LeafyGreen UI Kit's custom instance of Emotion"

File: packages/lib/src/index.ts [hunk 4]
```diff
@@ -1,7 +1,3 @@
-import emotion from './emotion';
-
-export const ccClassName = (...args: Array<string>) => args.join(' ');
```
Comment (DesignerDave) [inline packages/lib/src/index.ts]: End of an era 😂

File: packages/Checkbox/src/Checkbox.story.tsx (hunk 1 above)
Comment (brookescarlett) [reply packages/Checkbox/src/Checkbox.story.tsx]: 👍

File: packages/RadioBoxGroup/src/RadioBoxGroup.story.js (hunk 2 above)
Comment (brookescarlett) [reply packages/RadioBoxGroup/src/RadioBoxGroup.story.js]: 👍

File: packages/emotion/package.json (hunk 3 above)
Comment (brookescarlett) [reply packages/emotion/package.json]: 👍

File: packages/lib/src/index.ts (hunk 4 above)
Comment (brookescarlett) [reply packages/lib/src/index.ts]: Bye byeeeee 👋

**Pull request merged**